.gitignore

```diff
@@ -0,0 +1,2 @@
+*.swp
+.DS_Store
```

Comment:
FYI: this has been fixed in 7a782aa. The GitHub UI didn't like it when I rebased that one data file out.